frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -2193,10 +2193,14 @@ export const experimentLogic = kea<experimentLogicType>([
             },
         ],
         exposureCriteriaLabel: [
-            () => [],
-            (): string => {
-                // TODO: Implement exposure criteria label
-                return 'Default ($feature_flag_called)'
+            (s) => [s.experiment],
+            (experiment: Experiment): string => {
+                const exposureCriteria = experiment.exposure_criteria?.exposure_config
+                if (!exposureCriteria) {
+                    return 'Default ($feature_flag_called)'
+                }
+
+                return `Custom (${exposureCriteria.event})`
```

Comment:
I think that keeping this at the logic level is a bit overkill. This could be a function that takes and `Experiment` and returns a `string` at `Exposures` module level.

Reply:
Agree! 👍 

There is some cleanup to do in that logic file.